Update docs

jeremyhager opened the pull request:
- Add expandable gif demos for multiple cursor
- Add examples for themes to look at
- Add faq section with the original inspiration and idea of Lapce
- Add downloading from package manager page
- Add more development pages from `lapce/docs`
- Small whitespace fixes

jeremyhager committed:
updating a few docs

jeremyhager committed:
Expandable demos, Examples, faq and themes
- Add expandable gif demos for multiple cursor
- Add examples for themes to look at
- Small whitespace fixes
- Add faq section with the original inspiration and idea of Lapce

panekj commented on .gitbook/assets/add_next_occurrence.gif:
I'd prefer if all presented editors looked the same as the ones in docs

jeremyhager replied:
Makes sense, updated

panekj commented on development/architecture.md:
```diff
@@ -4,12 +4,12 @@
 ## Frontend
 
-The frontend uses [Druid](https://github.com/linebender/druid) for the GUI of Lapce. Druid is a data first GUI framework. And all the state and logic are in `lapce-data` sub crate, e.g., file buffers, editor tabs, cursor, etc. The UI widgets tree, layout, and rendering logic are in `lapce-ui` sub crate, including all the UI related stuff, e.g., keyboard and mouse events, file dialogs, context men ...
+The frontend uses [Floem](https://github.com/lapce/floem) for the GUI of Lapce. All the state and logic are in `lapce-data` sub crate, e.g., file buffers, editor tabs, cursor, etc. The UI widgets tree, layout, and rendering logic are in `lapce-ui` sub crate, including all the UI related stuff, e.g., keyboard and mouse events, file dialogs, context menus etc.
```
This needs to wait until we release floem version

jeremyhager replied:
Ah, my bad - fixed.

panekj commented on development/building-from-source.md:
I'd prefer not adding docs from repo

jeremyhager replied:
My thought with this was to add some docs for getting started on development. It felt like there needed to be more documentation for this, but I was concerned how it would keep up-to-date with the lapce/docs directory.

I could also create a simple page that points to docs in the Lapce directory - otherwise, if it really shouldn't be included at all I can remove all of those docs.

panekj commented on development/plugin-development.md:
```diff
@@ -14,4 +14,14 @@ Then in the plugin folder, run
 `volts publish`
 
-It should publish the plugin to the registry.&#x20;
+It should publish the plugin to the registry.
+
+## Examples
+There are lots of good examples from the community to base a plugin or theme. All can be found at [the regisry](https://plugins.lapce.dev/), but a few examples include:
+
+The [Template](https://github.com/lapce-community/lapce-volt-template) "plugin".  
+Be sure to include a README.md with links back to the repo!
+
+The [python](https://github.com/superlou/lapce-python) plugin.
+
+The [Go](https://github.com/lapce-community/lapce-go) plugin.
```
I'd prefer to not include this section

jeremyhager replied:
Alright, removed.

jeremyhager committed:
update demos, undo some docs
- gifs should be the whole Lapce window now
- Changed back to specifying Druid: Lapce still uses Druid vs. Floem
